content/guides/references/legacy-configuration.md

```diff
@@ -73,7 +73,6 @@ you should understand well. The default values listed here are meaningful.
 | `fileServerFolder`  | root project folder        | Path to folder where application files will attempt to be served from                                                                                                                                                                                                                      |
 | `fixturesFolder`    | `cypress/fixtures`         | Path to folder containing fixture files (Pass `false` to disable)                                                                                                                                                                                                                          |
 | `ignoreTestFiles`   | `*.hot-update.js`          | A String or Array of glob patterns used to ignore test files that would otherwise be shown in your list of tests. Cypress uses `minimatch` with the options: `{dot: true, matchBase: true}`. We suggest using [https://globster.xyz](https://globster.xyz) to test what files would match. |
-| `integrationFolder` | `cypress/integration`      | Path to folder containing integration test files                                                                                                                                                                                                                                           |
```

Comment:
Please revert this change

Reply:
Easy to get overzealous with these changes 😄